APCAppCore/APCAppCore/Library/Scoring/APCScoring.m

```diff
@@ -637,8 +637,8 @@ - (void)queryTaskId:(NSString *)taskId
     
     NSArray *tasks = [localContext executeFetchRequest:request error:&error];
     
-    for (APCScheduledTask *task in tasks) {
-        if ([task.completed boolValue]) {
+    for (APCTask *task in tasks) {
+        if (!task.taskFinished) {
```

Comment:
This looks like it reverses the logic of the prior implementation. Probably correct if it's working for you, but just making sure it's intentional?